sig-documentation/README.md

```diff
@@ -0,0 +1,43 @@
+# Proposal: Spinnaker Documentation Special Interest Group
+
+## Roles & Responsibilities of the SIG Leads
+
+### SIG Lead(s):
```

Comment:
cc @ezimanyi -- someone from your side?

Reply:
@dorbin: Is this something you'd be interested in being a part of?

Reply:
Yes, and in fact I was just in communication with Brian and with Ben Klein about this. I'm glad Brian's taking the lead on this, and I definitely want to be a part of it.